modules/acs-architecture-overview.adoc

```diff
@@ -7,18 +7,21 @@
 
 {product-title} ({product-title-short}) uses a distributed architecture that supports high-scale deployments and is optimized to minimize the impact on the underlying {ocp} or Kubernetes nodes.
 
-.{product-title} architecture for Kubernetes
-image::acs-architecture-kubernetes.png[{product-title} architecture for Kubernetes]
-
 [NOTE]
 ====
 The architecture is slightly different when you install {product-title-short} on Kubernetes and in {ocp}.
 However, the underlying components and the interactions between them remain the same.
 ====
 
+.{product-title-short} architecture for Kubernetes
+
+The following graphic shows the architecture with the StackRox Scanner, the default scanner for version 4.3 and earlier. For release 4.4, Scanner V4 installation is optional.
```

Comment:
Doesn't this sentence suggest that StackRox Scanner is not the default scanner anymore after 4.3? I have found the wording a bit weird here, because it is still the default scanner.